### DISCUSSION TITLE
Very urgent！

### DISCUSSION THREAD

**Discussion by richardlzx** [tensorflow/compression#112] [Q&A]
Hello everyone, I would like to ask what is the corresponding lamda of different quality (1-8) in the pretraining model of hypreprior?

**Comment by 0x00b1** [comment]
I am curious about this too!

**Comment by jonaballe** [comment]
Hi, the lambda values for MSE-trained models are `0.1 * 2**{-6..1}` and the values for MS-SSIM-trained models are `2**{-1..6}`. Hope this helps!

**Reply by 0x00b1** [reply]
Thanks @jonycgn!
